elastic/integrations
# [Kubernetes] Update namespace filter on data streams

## Proposed commit message
Some datastreams have present the `namespace` filter. I will update it to:

```yaml
      - name: namespace
        type: text
        title: Namespace
        description: Enrich events with metadata from the resources in this namespace. Empty value means all namespaces.
        multi: false
        required: false
        show_user: false
```

The namespace filter should be moved to the group level.

## Checklist





## How to test this PR locally


1. Build Kubernetes integration:
```bash
elastic-package build
```

2. Update registry:
```bash
elastic-package stack up --services package-registry
```

3. Install elastic agent with image 8.15.0-SNAPSHOT. It should use a policy with Kubernetes integration.


## Related issues

Relates https://github.com/elastic/beats/issues/38978


## Results

See https://github.com/elastic/integrations/pull/10213#issuecomment-2203027580.
